snuba/subscriptions/data.py

```diff
@@ -229,8 +243,17 @@ def add_conditions(
                 Literal(None, timestamp),
             ),
         ]
-
-        if offset is not None:
+        if self.organization:
+            conditions_to_add += [
+                binary_condition(
+                    ConditionFunctions.EQ,
+                    Column("_snuba_org_id", None, "org_id"),
```

Comment:
Why did you assign the `_snuba_org_id` alias?

Reply:
I basically mirrored this https://github.com/getsentry/snuba/blob/5390cf11ef086385be696e54d10d604a6a62de0a/snuba/query/organization_extension.py#L27

Reply:
Hmm, I think that's specific to extensions because of where they happen in the query pipeline (after aliases assigned). I don't think you need to manually assign any alias here, just leave it as None.